itou/users/adapter.py

```diff
@@ -48,19 +47,21 @@ def get_logout_redirect_url(self, request):
             if token:
                 params = {"token": token}
                 ic_base_logout_url = reverse("inclusion_connect:logout")
-                redirect_url = f"{ic_base_logout_url}?{urlencode(params)}"
+                return f"{ic_base_logout_url}?{urlencode(params)}"
         # France Connect
         fc_token = request.session.get(FRANCE_CONNECT_SESSION_TOKEN)
         fc_state = request.session.get(FRANCE_CONNECT_SESSION_STATE)
         if fc_token:
             params = {"id_token": fc_token, "state": fc_state}
             fc_base_logout_url = reverse("france_connect:logout")
-            redirect_url = f"{fc_base_logout_url}?{urlencode(params)}"
+            return f"{fc_base_logout_url}?{urlencode(params)}"
         # PE Connect
-        if getattr(request.user, "kind", None) == IdentityProvider.PE_CONNECT:
-            peamu_id_token = self.socialaccount_set.filter(provider="peamu").get().extra_data["id_token"]
-            params = {"id_token_hint": peamu_id_token, "redirect_uri": get_absolute_url(redirect_url)}
-            redirect_url = f"{settings.PEAMU_AUTH_BASE_URL}/compte/deconnexion?{urlencode(params)}"
+        pe_token = request.session.get(FRANCE_CONNECT_SESSION_TOKEN)
+        pe_state = request.session.get(FRANCE_CONNECT_SESSION_STATE)
```

Comment:
```suggestion
        pe_token = request.session.get(PE_CONNECT_SESSION_TOKEN)
        pe_state = request.session.get(PE_CONNECT_SESSION_STATE)
```

Peut-être ajouter un test pour couvrir ce cas ?